app/models/about_us_info.rb

```diff
@@ -0,0 +1,3 @@
+class AboutUsInfo < ApplicationRecord
+  has_many :employees
+end
```

Comment:
This is a independent table, We will not add any reference here..

Reply:
I was referencing [https://www.searchenginejournal.com/about/](url) this website and was thinking about showing the employs.

Reply:
@Dhyey54 Unable to open this URL. Can you please check once.